zerver/openapi/zulip.yaml

```diff
@@ -14373,10 +14373,12 @@ components:
         since gravatar URLs are long, random strings and thus do not
         compress well. The `client_gravatar` field is set to `true` if
         clients can compute their own gravatars.
+
+        **Changes**: This parameter now defaults to `true` after Zulip 5.0 (feature level 91).
```

Comment:
nit: Do you mean it defaults to `true` in 5.0+, or in 5.1+? I think you might mean "starting in Zulip 5.0" or something like that.

Reply:
Yeah, "starting 5.0". I'll reword that.

Reply:
The way we write these documentation should always be that the first section explain the current state, and the Changes sections(s) explain how the previous logic worked; so we never want to say "now defaults".  That language would need to be rewritten if we added a newer change to the documentation.